homeassistant/components/ialarm/config_flow.py

```diff
@@ -0,0 +1,64 @@
+"""Config flow for Antifurto365 iAlarm integration."""
+import logging
+
+from pyialarm import IAlarm
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.const import CONF_HOST, CONF_PIN, CONF_PORT
```

Comment:
```suggestion
from homeassistant.const import CONF_HOST, CONF_PORT
```

It doesn't look like the device requires a code.